.github/workflows/ci_linux_mu4.yml

```diff
@@ -79,6 +79,8 @@ jobs:
           DO_PUBLISH='false'
         fi
 
+        ARTIFACT_NAME="$(tr "\":<>|*?\\/" "_" <<<"MU4_${{ github.run_id }}_Lin_${{ github.event.pull_request.number }}_${{ github.event.pull_request.title }}")"
```

Comment:
The build scripts use a variable called `ARTIFACT_NAME`, this is the name of the executable file.
There may be a conflict with this name.
So, I propose to rename, for example, to `UPLOAD_ARTIFACT_NAME`

Reply:
Done!